src/transformStateWithClones.js

```diff
@@ -7,7 +7,40 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let newObj = {
+    ...state,
+  };
+  const newArray = [];
```

Comment:
work on your variable names, make it more specific, `newArray` of what exactly? `stateClones` would be a great fit here